Modules/Core/Common/test/itkPointSetToImageFilterTest1.cxx

```diff
@@ -0,0 +1,116 @@
+/*=========================================================================
+ *
+ *  Copyright NumFOCUS
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0.txt
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *=========================================================================*/
+
+#include "itkImageFileWriter.h"
+#include "itkPointSetToImageFilter.h"
+#include "itkPointSet.h"
+#include "itkTestingMacros.h"
+
+
+int
+itkPointSetToImageFilterTest1(int argc, char * argv[])
+{
+  if (argc != 2)
+  {
+    std::cerr << "Missing parameters." << std::endl;
+    std::cerr << "Usage: " << itkNameOfTestExecutableMacro(argv);
+    std::cerr << " outputImageFile" << std::endl;
+    return EXIT_FAILURE;
+  }
+
+  constexpr unsigned int PointSetDimension = 2;
+  constexpr unsigned int ImageDimension = 2;
+
+  using PointSetPointType = float;
+  using PixelType = unsigned char;
+
+  using PointSetType = itk::PointSet<PointSetPointType, PointSetDimension>;
+  using BinaryImageType = itk::Image<PixelType, ImageDimension>;
+
+  using PointSetType = itk::PointSet<PointSetPointType, PointSetDimension>;
+
+  using PointType = PointSetType::PointType;
+
+  auto pointSet = PointSetType::New();
+
+  // Create a point set describing a circle
+  float        radius = 100.0;
+  unsigned int count = 0;
+  for (float theta = 0; theta < 2.0 * itk::Math::pi; theta += 0.1)
+  {
+    PointType point;
+    point[0] = radius * std::cos(theta);
+    point[1] = radius * std::sin(theta);
+
+    pointSet->SetPoint(count, point);
+    count++;
+  }
```

Comment:
This was borrowed from other examples/tests, e.g.https://github.com/InsightSoftwareConsortium/ITK/blob/d715b098ba992404af994d7d009a066c42f8c7f7/Modules/Registration/Metricsv4/test/itkExpectationBasedPointSetMetricRegistrationTest.cxx

For future uses, best would be to relocate such parts to a testing utils module where they can be re-used across the code base. It is left for a separate PR.